<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="local" default="false"/>
	<group unless="$(arg local)">
		<!-- multimaster_fkie -->
		<node name="$(anon master_discovery)" pkg="master_discovery_fkie" type="master_discovery" output="log">
			<rosparam param="robot_hosts">[donny,mikey,leo,raph,tams223]</rosparam>
		</node>
		<node name="$(anon master_sync)" pkg="master_sync_fkie" type="master_sync">
			<rosparam param="sync_services">['/rpc_score_master']</rosparam>
<rosparam param="sync_topics">
- /rpc_game/alive
- /rpc_game/gamestate
- /rpc_game/submissions
</rosparam>
		</node>
	</group>

	<node pkg="rpc_game_client" name="$(anon game_client)" type="game_client.py" output="screen" />
</launch>
